mitodl/ocw-hugo-themes
# Show featured course list in each collection if it exists

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #551

#### What's this PR do?
Shows a carousel of the collection's featured list courses if it has a featured list.

#### How should this be manually tested?
- Run `npm run start:www`
- Verify that the home page still shows featured courses from the `featured-courses-homepage` list, with a black 'Featured Courses' title.
- Go to each collection page and verify that the featured courses show up (if there is a featured list for the collection), with a red 'Featured Courses` title.

#### Screenshots (if appropriate)
***With cover image and featured courses***
![Screenshot 2022-03-31 at 10-26-17 Energy MIT OpenCourseWare Free Online Course Materials](https://user-images.githubusercontent.com/187676/161079593-abcd11b0-4bc3-4135-b702-ba043fb4cda6.png)

***Without cover image and featured courses***
![Screenshot 2022-03-31 at 10-29-22 Test Collection MIT OpenCourseWare Free Online Course Materials](https://user-images.githubusercontent.com/187676/161079591-780c9984-6c44-4d8b-b628-a2963b011881.png)
